src/__stories__/button-group-story.tsx

```diff
@@ -10,51 +10,40 @@ export default {
 const handleOnPress = () => window.alert('pressed!');
 
 type Args = {
```

Comment:
include new boolean control to change to small all the buttons, pls

Reply:
done